src/zope/component/_api.py

```diff
@@ -14,22 +14,13 @@
 """Zope 3 Component Architecture
 """
 import sys
-import types
 
 from zope.hookable import hookable
 from zope.interface import Interface
-from zope.interface import implementedBy
-from zope.interface import providedBy
 
-from zope.component.interfaces import IComponentArchitecture
-from zope.component.interfaces import IComponentRegistrationConvenience
 from zope.component.interfaces import IFactory
-from zope.component.interfaces import ComponentLookupError
-from zope.component.interfaces import IComponentLookup
-from zope.component._compat import _BLANK
-from zope.component._declaration import adaptedBy
-from zope.component._declaration import adapter
-from zope.component._declaration import adapts
+from zope.interface.interfaces import ComponentLookupError
+from zope.interface.interfaces import IComponentLookup
```

Comment:
Import ordering: stdlib (sys), 3rd-party (zope.hookable, zope.interface), then zope.component.

Reply:
That's twice in two days that I've been bitten by import groups in reviews. I'm beginning to appreciate Plone's policy of [just ignoring that](https://docs.plone.org/develop/styleguide/python.html#grouping-and-sorting) 😉